dask_kubernetes/core.py

```diff
@@ -205,6 +206,9 @@ def __init__(self, idle_timeout: str, **kwargs):
             self.service_template.spec.type = dask.config.get(
                 "kubernetes.scheduler-service-type"
             )
+        self._service_wait_timeout_s = (
+            dask.config.get("kubernetes.scheduler-service-wait-timeout", 30) or 30
+        )
```

Comment:
We should move this to the [`KubeCluster`](https://github.com/dask/dask-kubernetes/blob/master/dask_kubernetes/core.py#L381) object and allow folks to pass it as a kwarg there too.

You would then add it to the scheduler spec [here](https://github.com/dask/dask-kubernetes/blob/master/dask_kubernetes/core.py#L489) which would then get passed into the `Scheduler` object [here](https://github.com/dask/dask-kubernetes/blob/master/dask_kubernetes/core.py#L136).

Reply:
I actually started by doing what you propose, but seeing the already numerous kwargs in the `KubeCluster` constructor, I ended up adding it only in the configuration since I wasn't sure about adding yet another argument.

To be sure I understood correctly. Should I move everything entirely or should I add the "constructor way" to set it in addition to the possibility of using the config ?

Reply:
Yes we should use the constructor and config for all options. This class has a lot of kwargs and we will certainly add more in the future. It's better to be consistent.

Reply:
Ok sounds good to me. I'll make the modifications.